2021-07-05 00:00:30,915 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:55,352 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,747 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,751 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,755 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,758 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,762 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,766 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,769 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,773 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,777 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,781 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,785 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,788 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,792 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,795 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,798 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,802 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,806 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,810 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,813 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,820 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,823 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,826 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,831 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,836 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,839 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,843 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,846 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,849 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,854 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,858 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,861 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,864 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,868 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,871 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,874 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,878 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,881 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,885 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,888 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,891 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,894 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,898 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,901 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,906 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,910 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,913 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:00:57,917 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:36,475 - INFO - ServiceNode - [EtcdSupport.py:549]: we have create ServiceRegister
2021-07-05 00:25:36,477 - INFO - ServiceNode - [EtcdSupport.py:552]: we have create ServiceFinder
2021-07-05 00:25:36,485 - INFO - TcpServer - [TcpServer.py:100]: TCP Server on ('127.0.0.1', 9201)
2021-07-05 00:25:39,484 - INFO - ServiceFinder - [EtcdSupport.py:382]: _init_info, now_url: http://127.0.0.1:2379/v2/keys?recursive=true
2021-07-05 00:25:39,489 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> dispatcher_service: ('127.0.0.1', 9100)
2021-07-05 00:25:39,490 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> lobby_gate: ('127.0.0.1', 9201)
2021-07-05 00:25:39,492 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> lobby_server: ('127.0.0.1', 10001)
2021-07-05 00:25:39,493 - INFO - ServiceFinder - [EtcdSupport.py:302]: init service info from etcd success, will going to watch, at etcd index -> 5953
2021-07-05 00:25:40,637 - INFO - TcpProtocol - [ConnMgr.py:49]: TCP ROLE_TYPE_PASSIVE peer_addr=('127.0.0.1', 14403) is connected !!!!
2021-07-05 00:25:40,676 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 10001) is connected !!!!
2021-07-05 00:25:40,679 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:40,697 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:40,700 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:40,703 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:43,684 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:43,688 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:43,691 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:43,694 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:46,523 - INFO - TcpProtocol - [ConnMgr.py:49]: TCP ROLE_TYPE_PASSIVE peer_addr=('127.0.0.1', 14410) is connected !!!!
2021-07-05 00:25:46,552 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:46,562 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14410)
2021-07-05 00:25:46,565 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14410)
2021-07-05 00:25:46,569 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:46,689 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:46,695 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:46,699 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:46,702 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:49,548 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:49,553 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14410)
2021-07-05 00:25:49,557 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14410)
2021-07-05 00:25:49,561 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:49,685 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:49,690 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:49,693 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:49,697 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:52,555 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:52,560 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14410)
2021-07-05 00:25:52,563 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14410)
2021-07-05 00:25:52,567 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:52,694 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:52,699 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:52,702 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:52,705 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:55,576 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:55,580 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14410)
2021-07-05 00:25:55,583 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14410)
2021-07-05 00:25:55,588 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:55,702 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:55,706 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:55,709 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:55,712 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:58,577 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:58,583 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14410)
2021-07-05 00:25:58,588 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14410)
2021-07-05 00:25:58,591 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:58,701 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 00:25:58,706 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:58,710 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 14403)
2021-07-05 00:25:58,714 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 10001)
2021-07-05 01:05:13,558 - INFO - ServiceNode - [EtcdSupport.py:549]: we have create ServiceRegister
2021-07-05 01:05:13,559 - INFO - ServiceNode - [EtcdSupport.py:552]: we have create ServiceFinder
2021-07-05 01:05:13,567 - INFO - TcpServer - [TcpServer.py:100]: TCP Server on ('127.0.0.1', 9201)
2021-07-05 01:05:16,582 - INFO - ServiceFinder - [EtcdSupport.py:382]: _init_info, now_url: http://127.0.0.1:2379/v2/keys?recursive=true
2021-07-05 01:05:16,585 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> lobby_server: ('127.0.0.1', 10001)
2021-07-05 01:05:16,588 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> dispatcher_service: ('127.0.0.1', 9100)
2021-07-05 01:05:16,588 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> lobby_gate: ('127.0.0.1', 9201)
2021-07-05 01:05:16,589 - INFO - ServiceFinder - [EtcdSupport.py:302]: init service info from etcd success, will going to watch, at etcd index -> 7197
2021-07-05 01:05:33,414 - INFO - TcpProtocol - [ConnMgr.py:49]: TCP ROLE_TYPE_PASSIVE peer_addr=('127.0.0.1', 5186) is connected !!!!
2021-07-05 01:05:33,439 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:05:33,443 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 10001) is connected !!!!
2021-07-05 01:05:47,491 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\ConnBase.py:192"]: peer addr: ('127.0.0.1', 5186), close_reason='[WinError 64] The specified network name is no longer available'
2021-07-05 01:07:48,629 - INFO - TcpProtocol - [ConnMgr.py:49]: TCP ROLE_TYPE_PASSIVE peer_addr=('127.0.0.1', 5203) is connected !!!!
2021-07-05 01:07:48,669 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:08:00,550 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\ConnBase.py:192"]: peer addr: ('127.0.0.1', 5203), close_reason='[WinError 64] The specified network name is no longer available'
2021-07-05 01:15:56,120 - INFO - TcpProtocol - [ConnMgr.py:49]: TCP ROLE_TYPE_PASSIVE peer_addr=('127.0.0.1', 5240) is connected !!!!
2021-07-05 01:15:56,146 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:20:08,836 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\ConnBase.py:192"]: peer addr: ('127.0.0.1', 5240), close_reason='[WinError 64] The specified network name is no longer available'
2021-07-05 01:20:22,292 - INFO - TcpProtocol - [ConnMgr.py:49]: TCP ROLE_TYPE_PASSIVE peer_addr=('127.0.0.1', 5268) is connected !!!!
2021-07-05 01:20:22,318 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:20:30,590 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\ConnBase.py:192"]: peer addr: ('127.0.0.1', 5268), close_reason='[WinError 64] The specified network name is no longer available'
2021-07-05 01:22:43,376 - INFO - ServiceNode - [EtcdSupport.py:549]: we have create ServiceRegister
2021-07-05 01:22:43,377 - INFO - ServiceNode - [EtcdSupport.py:552]: we have create ServiceFinder
2021-07-05 01:22:43,384 - INFO - RudpServer - [RudpServer.py:64]: RUDP Server on ('127.0.0.1', 9201)
2021-07-05 01:22:46,392 - INFO - ServiceFinder - [EtcdSupport.py:382]: _init_info, now_url: http://127.0.0.1:2379/v2/keys?recursive=true
2021-07-05 01:22:46,409 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> dispatcher_service: ('127.0.0.1', 9100)
2021-07-05 01:22:46,410 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> lobby_gate: ('127.0.0.1', 9201)
2021-07-05 01:22:46,411 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> lobby_server: ('127.0.0.1', 10001)
2021-07-05 01:22:46,413 - INFO - ServiceFinder - [EtcdSupport.py:302]: init service info from etcd success, will going to watch, at etcd index -> 7820
2021-07-05 01:25:16,755 - INFO - RudpProtocol - [ConnMgr.py:111]: RUDP ROLE_TYPE_PASSIVE peer_addr=('127.0.0.1', 65279) is connected !!!!
2021-07-05 01:25:16,800 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:25:18,855 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:25:19,445 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 10001) ... 1
2021-07-05 01:25:21,461 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:25:22,052 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 10001) ... 2
2021-07-05 01:25:24,091 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:25:24,686 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 10001) ... 3
2021-07-05 01:25:26,739 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:25:27,334 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 10001) ... 4
2021-07-05 01:25:29,391 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:25:29,999 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 10001) ... 5
2021-07-05 01:25:32,054 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:25:32,647 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 10001) ... 6
2021-07-05 01:25:32,648 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:65"]: try 6 times , still can't connect tcp remote addr: ('127.0.0.1', 10001)
2021-07-05 01:25:32,787 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:25:33,797 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:25:34,793 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:25:35,781 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:25:36,785 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:25:37,787 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:25:38,799 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:30:39,873 - INFO - ServiceNode - [EtcdSupport.py:549]: we have create ServiceRegister
2021-07-05 01:30:39,875 - INFO - ServiceNode - [EtcdSupport.py:552]: we have create ServiceFinder
2021-07-05 01:30:39,883 - INFO - RudpServer - [RudpServer.py:64]: RUDP Server on ('127.0.0.1', 9201)
2021-07-05 01:30:42,887 - INFO - ServiceFinder - [EtcdSupport.py:382]: _init_info, now_url: http://127.0.0.1:2379/v2/keys?recursive=true
2021-07-05 01:30:42,904 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> dispatcher_service: ('127.0.0.1', 9100)
2021-07-05 01:30:42,905 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> lobby_gate: ('127.0.0.1', 9201)
2021-07-05 01:30:42,907 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> lobby_server: ('127.0.0.1', 10001)
2021-07-05 01:30:42,908 - INFO - ServiceFinder - [EtcdSupport.py:302]: init service info from etcd success, will going to watch, at etcd index -> 7931
2021-07-05 01:30:43,362 - INFO - RudpProtocol - [ConnMgr.py:111]: RUDP ROLE_TYPE_PASSIVE peer_addr=('127.0.0.1', 58900) is connected !!!!
2021-07-05 01:30:43,410 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
2021-07-05 01:30:43,416 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 10001) is connected !!!!
2021-07-05 01:31:09,583 - INFO - RudpProtocol - [ConnMgr.py:111]: RUDP ROLE_TYPE_PASSIVE peer_addr=('127.0.0.1', 58902) is connected !!!!
2021-07-05 01:31:09,617 - DEBUG - ProxyLobbyRpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 10001)
